src/main/scala/org/bitcoins/core/gen/TransactionGenerators.scala

```diff
@@ -219,7 +219,7 @@ trait TransactionGenerators extends BitcoinSLogger {
     (creditingTx,outputIndex) = buildCreditingTransaction(p2shScriptSig.redeemScript, wtxSigComponent.amount)
     (signedTx,inputIndex) = buildSpendingTransaction(creditingTx,p2shScriptSig,outputIndex,witness)
     signedTxSignatureComponent = WitnessV0TransactionSignatureComponent(signedTx,inputIndex,
-      p2shScriptPubKey, Policy.standardScriptVerifyFlags, wtxSigComponent.amount)
+      p2shScriptPubKey, Policy.standardScriptVerifyFlags, wtxSigComponent.amount, SigVersionWitnessV0)
```

Comment:
Instead of passing the constant `SiversionWitnesV0` we need to call `BitcoinScriptUtil.parseSigVersion`